react/src/components/ComputeSessionNodeItems/ContainerLogModalWithLazyQueryLoader.tsx

```diff
@@ -0,0 +1,50 @@
+import { useCurrentProjectValue } from '../../hooks/useCurrentProject';
+import ContainerLogModal from './ContainerLogModal';
+import { ContainerLogModalWithLazyQueryLoaderQuery } from './__generated__/ContainerLogModalWithLazyQueryLoaderQuery.graphql';
+import graphql from 'babel-plugin-relay/macro';
+import { useState } from 'react';
+import { useLazyLoadQuery } from 'react-relay';
+
+const ContainerLogModalWithLazyQueryLoader: React.FC<{
+  sessionId: string;
+  afterClose?: () => void;
+}> = ({ sessionId, afterClose }) => {
+  const currentProject = useCurrentProjectValue();
+  const [open, setOpen] = useState(true);
+  const { compute_session_node } =
+    useLazyLoadQuery<ContainerLogModalWithLazyQueryLoaderQuery>(
+      graphql`
+        query ContainerLogModalWithLazyQueryLoaderQuery(
+          $sessionId: GlobalIDField!
+          $project_id: UUID!
+        ) {
+          compute_session_node(id: $sessionId, project_id: $project_id) {
+            ...ContainerLogModalFragment
+          }
+        }
+      `,
+      {
+        sessionId,
+        project_id: currentProject.id,
+      },
+    );
+
+  return (
+    compute_session_node && (
+      <ContainerLogModal
```

Comment:
When we control the modal render with compute_session_node, the animation that opens the modal doesn't work, is there any way to fix it? I've tried several ways to fix it, but failed.

Reply:
`ContainerSessionLogModal` is implemented using GraphQL fragments, but due to the existing session launcher implemented with Lit, we had to implement `ContainerLogModalWithLazyQueryLoader` to fetch GraphQL data using lazy loading. Although there are shortcomings in the animation handling you mentioned. Since `ContainerLogModalWithLazyQueryLoader` is a temporary workaround, I believe it is acceptable to merge at this level in this PR.